releases/release-9.0.0.md

```diff
@@ -0,0 +1,355 @@
+
+    In v9.0.0, the `execution info` output is optimized for clearer representation of each metric. For example, `time` now refers to the wall-clock time for operator execution, `loops` indicates how many times the current operator is called by its parent operator, and `total_time` represents the cumulative duration of all concurrent executions. These optimizations help you better understand the SQL execution process and devise more targeted optimization strategies.
+
+    For more information, see [documentation](/sql-statements/sql-statement-explain-analyze.md).
+
+### Security
+
+### Data migration
+
+* Support query argument redaction in DM logs [#11489](https://github.com/pingcap/tiflow/issues/11489) @[db-will](https://github.com/db-will) **tw@Oreoxmt** <!--2030-->
+
+    Starting from v9.0.0, you can use the `redact-info-log` configuration item to enable the DM log redaction feature. When enabled, query arguments that contain sensitive data in DM logs are replaced with the `?` placeholder. To enable this feature, set `redact-info-log` to `true` in the DM-worker configuration file or pass `--redact-info-log=true` when starting DM. This feature only desensitizes query arguments, not the entire SQL statement, and requires a DM-worker restart to take effect.
+
+    For more information, see [documentation](/dm/dm-worker-configuration-file.md#redact-info-log-new-in-v900).
+
+* TiDB Lightning supports compatibility with `sql_require_primary_key=ON` in TiDB [#57479](https://github.com/pingcap/tidb/issues/57479) @[lance6716](https://github.com/lance6716) **tw@Oreoxmt** <!--2026-->
+
+    When you enable the system variable [`sql_require_primary_key`](/system-variables.md#sql_require_primary_key-new-in-v630) in TiDB, tables must have a primary key. To prevent table creation failures, TiDB Lightning adds a default primary key to its internal error-logging and conflict-detection tables (`conflict_error_v4`, `type_error_v2`, and `conflict_records_v2`). If you have automation scripts that use these internal tables, update them to accommodate the new schema, which includes the primary key.
+
+* Migrate sync-diff-inspector from `pingcap/tidb-tools` to `pingcap/tiflow` repository [#11672](https://github.com/pingcap/tiflow/issues/11672) @[joechenrh](https://github.com/joechenrh) **tw@Oreoxmt** <!--2070-->
+
+    Starting from v9.0.0, the sync-diff-inspector tool is moved from the [`pingcap/tidb-tools`](https://github.com/pingcap/tidb-tools) repository to [`pingcap/tiflow`](https://github.com/pingcap/tiflow). This change unifies replication and migration tools ([DM](/dm/dm-overview.md), [TiCDC](/ticdc/ticdc-overview.md), and [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)) into a single repository.
```

Comment:
```suggestion
    Starting from v9.0.0, the sync-diff-inspector tool is moved from the [`pingcap/tidb-tools`](https://github.com/pingcap/tidb-tools) repository to [`pingcap/tiflow`](https://github.com/pingcap/tiflow). With this change, sync-diff-inspector is now maintained in the same repository as [DM](/dm/dm-overview.md) and [TiCDC](/ticdc/ticdc-overview.md), which unifies the management of these replication and migration tools.

Reply:
reason: "This change unifies replication and migration tools ([DM](/dm/dm-overview.md), [TiCDC](/ticdc/ticdc-overview.md), and [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)) into a single repository." is confusing, which might the misunderstanding that the repository of DM of TiCDC is changed in v9.0.0 too